src/server/routes/launchpad.js

```diff
@@ -14,6 +15,9 @@ router.use('/launchpad/snaps', json());
 router.post('/launchpad/snaps', newSnap);
 router.get('/launchpad/snaps', findSnap);
 
+router.use('/launchpad/snaps/list', json());
+router.get('/launchpad/snaps/list', findSnaps);
```

Comment:
I wasn't sure about the route here as already existing`GET /launchpad/snaps` seems to fit nice here, but I didn't want to break the compatibility of previous calls for now.